styleguide.md

```diff
@@ -0,0 +1,76 @@
+This Component Library demonstrates styles and components for **HMRC** applications. It is built on top of the [GOV.UK frontend toolkit](https://github.com/alphagov/govuk_frontend_toolkit) and the [GOV.UK template](https://github.com/alphagov/govuk_template) and it uses the [hmrc-component-library-template](https://github.com/hmrc/component-library-template).
+  
+  
+* [What is the Component Library](#what-is-the-component-library)
+* [Why do we need a Component Library?](#why-do-we-need-a-component-library)
+* [Adding Components](#adding-components)
+* [Discussing Changes](#discussing-changes)
+* [Reporting Issues / Actioning work - Components](#reporting-issues)
+* [Development details](#development-details)
+* [Component Library Template](#component-library-template)
+* [Reporting Issues / Actioning work - Template](#reporting-issues-template)
+* [Developing on the template](#developing-on-the-template)
+  
+  
+<a name="what-is-the-component-library"></a>
+## What is the Component Library?
+The Component Library is generated output of [HMRC assets-fronted](https://github.com/hmrc/assets-frontend) stylesheets. 
+
+* It allows Designers and Developers to see what Components we have across our Service
```

Comment:
I'd argue that this is the only point from this and the next section that need to be on this page.

How we/designers communicate/work is an internal thing and doesn't effect anyone outside HMRC who might be looking at the component library